tests/plugins/easyimage/manual/sideimagebig.html

```diff
@@ -0,0 +1,97 @@
+<h2>Classic editor</h2>
```

Comment:
This test is pretty much broken on IE11. Table resize does not work here,

Reply:
Looks like it is not the issue with added styles, but basically an issue with IE11 and easyimage widget inside table. Without `min-width` and even `max-width` table is still not resizable.

Reply:
That's an interesting issue, could you extract this? I think that we simply should change manual test concept here to have it solved.

Reply:
Change the test to use smaller image so resizing is possible (but image still scales) and add proper info.